utils/keys.go

```diff
@@ -117,53 +116,47 @@ func ImportKeys(from io.Reader, to []Importer, fallbackRole string, fallbackGun
 				}
 			}
```

Comment:
Need to `block.Headers["path"] = filepath.Base(rawPath)` otherwise later code will incorrectly use the legacy style path.